### PR TITLE
Cronjob Scheduling Send mail weekly

### PR DESCRIPTION
**Pull request by phamviethieu**
Cronjob Scheduling Send mail weekly

## Related Tickets:
- [#25766](https://edu-redmine.sun-asterisk.vn/issues/25766)

## WHAT this PR does?
Cronjob sheduling mail weekly at 16:00 Sunday

## Notes Impacted Areas:
- N/A.

## Attachments:
Peek 2020-07-22 08-44

### CODE REVIEW

**Comment by phamviethieu** [comment]
ready

File: app/Console/Commands/BookingCron.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+        $bookingsWaiting = Booking::whereBetween('created_at', [$startOfWeek, $now])
+            ->where('status', config('status.booking_status.waiting'))
+            ->count();
+        $bookingsApproved = Booking::whereBetween('created_at', [$startOfWeek, $now])
+            ->where('status', config('status.booking_status.approved'))
+            ->count();
+        $bookingsCanceled = Booking::whereBetween('created_at', [$startOfWeek, $now])
+            ->where('status', config('status.booking_status.canceled'))
+            ->count();
```
Comment (huyenttm-0901) [inline app/Console/Commands/BookingCron.php]: dùng repo

File: app/Console/Commands/BookingCron.php [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+        $users = User::where('role_id', config('role.admin'))->get();
+        foreach ($users as $user) {
+            $data = [
+                'name' => $user->name,
+                'info' => $user->phone_number,
+            ];
+            Mail::to($user->email)->send(new BookingReport($data, $bookings));
+        }
+        dd("success");
```
Comment (huyenttm-0901) [inline app/Console/Commands/BookingCron.php]: sao lại dd ở đây?
```suggestion
        $this->info('success);
```

**Comment by phamviethieu** [comment]
ready

**Pull request merged**